### PR TITLE
Do something weird here just to demonstrate github

### PR DESCRIPTION
**Pull request by lpereira**

### CODE REVIEW

**Comment by fvincenzo** [comment]
Add me to the review!!!

**Review by nashif — COMMENTED** [review]
this is nice

**Comment by nashif** [comment]
nice

**Comment by nashif** [comment]
nice

File: kernel/microkernel/k_semaphore.c [hunk 1]
```diff
@@ -112,34 +112,7 @@ void _k_sem_group_wait_cancel(struct k_args *A)
...
-
-/*
- * Determine if the wait cancellation request is being
- * processed after the state of the 'waiters' packet state
- * has been updated to _K_SVC_SEM_GROUP_WAIT_READY, but before
- * the _K_SVC_SEM_GROUP_WAIT_READY packet has been processed.
- * This will occur if a _K_SVC_SEM_GROUP_WAIT_TIMEOUT
- * timer expiry occurs between the update of the packet state
- * and the processing of the WAITMRDY packet.
- */
```
Comment (galak) [inline kernel/microkernel/k_semaphore.c:131]: I think this should be added back

**Comment by MaureenHelm** [comment]
hello!

**Pull request merged**